src/RoomsSDKAdapter.integration.test.js

```diff
@@ -1,42 +1,44 @@
 import {createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
-import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+import WebexSDKAdapter from './';
 
 describe('Rooms SDK Adapter', () => {
-  let roomsSDKAdapter, subscription, user;
+  let createdRoom, getRoom$, webexSDKAdapter, subscription, user;
 
   // Since these are integration tests with live data,
   // increase the async "idle" timeout so jest doesn't error early.
   jest.setTimeout(30000);
 
   beforeAll(async () => {
     user = await createIntegrationTestUser();
-  });
-
-  beforeEach(() => {
-    roomsSDKAdapter = new RoomsSDKAdapter(user.sdk);
+    webexSDKAdapter = new WebexSDKAdapter(user.sdk);
+    await webexSDKAdapter.connect();
   });
 
   afterAll(async () => {
-    // This is done after unsubscribing, but since unsubscribe can't be async, manually call it.
-    await user.sdk.rooms.stopListening();
-
     try {
       await removeIntegrationTestUser(user);
+      await webexSDKAdapter.disconnect();
     } catch (reason) {
       // eslint-disable-next-line no-console
       console.warn('Failed to delete test user for Room SDK Adapter integration tests', reason);
     }
   });
 
   describe('getRoom() returns', () => {
+    beforeEach(async () => {
+      createdRoom = await user.sdk.rooms.create({title: 'Webex Teams Test Room'});
+      getRoom$ = webexSDKAdapter.roomsAdapter.getRoom(createdRoom.id);
+    });
+
     afterEach(() => {
+      createdRoom = null;
```

Comment:
Should we actually delete from the platform? This should also happen at the end of the suite run

Reply:
not sure how to delete the object. I could either nullify or `undefined` them.